rllib/evaluation/rollout_worker.py

```diff
@@ -911,9 +911,8 @@ def sample(self) -> SampleBatchType:
         self.output_writer.write(batch)
 
         # Do off-policy estimation, if needed.
-        if self.reward_estimators:
```

Comment:
reward_estimators is already an empty list.

Reply:
@kouroshHakha Yes, but this fails downstream if postprocessing expects the train_results to be a dict of {policy_id: policy_results}. The other option is to add the OPE losses under the policy_results.